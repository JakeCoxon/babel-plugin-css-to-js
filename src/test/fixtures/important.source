css`
  foo {
    padding-top: 0px !important;
  }
`;
