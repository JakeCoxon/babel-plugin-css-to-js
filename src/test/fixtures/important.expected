({
  "foo": {
    paddingTop: "0px !important"
  }
});
